src/core/metadata/qgslayermetadataformatter.cpp

```diff
@@ -107,50 +107,53 @@ QString QgsLayerMetadataFormatter::contactsSectionHtml() const
+      if ( mMetadata.crs().isGeographic() )
+        myMetadata += QObject::tr( "Geographic" );
+      else
+        myMetadata += QObject::tr( "Projected" );
     }
-    myMetadata += QStringLiteral( "<strong>" ) + QObject::tr( "CRS" ) + QStringLiteral( ": </strong>" ) + spatialExtent.extentCrs.authid() + QStringLiteral( " - " );
-    myMetadata += spatialExtent.extentCrs.description() + QStringLiteral( " - " );
-    if ( spatialExtent.extentCrs.isGeographic() )
-      myMetadata += QObject::tr( "Geographic" );
-    else
-      myMetadata += QObject::tr( "Projected" );
-    myMetadata += QStringLiteral( "<br />" );
-    myMetadata += QStringLiteral( "<strong>" ) + QObject::tr( "X Minimum" ) + QStringLiteral( ": </strong>" ) +  qgsDoubleToString( spatialExtent.bounds.xMinimum() ) + QStringLiteral( "<br />" );
-    myMetadata += QStringLiteral( "<strong>" ) + QObject::tr( "Y Minimum" ) + QStringLiteral( ": </strong>" ) +  qgsDoubleToString( spatialExtent.bounds.yMinimum() ) + QStringLiteral( "<br />" );
-    myMetadata += QStringLiteral( "<strong>" ) + QObject::tr( "X Maximum" ) + QStringLiteral( ": </strong>" ) +  qgsDoubleToString( spatialExtent.bounds.xMaximum() ) + QStringLiteral( "<br />" );
-    myMetadata += QStringLiteral( "<strong>" ) + QObject::tr( "Y Maximum" ) + QStringLiteral( ": </strong>" ) +  qgsDoubleToString( spatialExtent.bounds.yMaximum() ) + QStringLiteral( "<br />" );
-    if ( spatialExtent.bounds.zMinimum() || spatialExtent.bounds.zMaximum() )
+    myMetadata += QLatin1String( "</td></tr>\n" );
+
+    myMetadata += QStringLiteral( "<tr><td class=\"highlight\">" ) + QObject::tr( "Spatial Extent" ) + QStringLiteral( "</td><td>" );
+    const QList< QgsLayerMetadata::SpatialExtent > spatialExtents = extent.spatialExtents();
+    for ( const QgsLayerMetadata::SpatialExtent &spatialExtent : spatialExtents )
     {
-      myMetadata += QStringLiteral( "<strong>" ) + QObject::tr( "Z Minimum" ) + QStringLiteral( ": </strong>" ) +  qgsDoubleToString( spatialExtent.bounds.zMinimum() ) + QStringLiteral( "<br />" );
-      myMetadata += QStringLiteral( "<strong>" ) + QObject::tr( "Z Maximum" ) + QStringLiteral( ": </strong>" ) +  qgsDoubleToString( spatialExtent.bounds.zMaximum() );
+      if ( notFirstRow )
+      {
+        myMetadata += QLatin1String( "<br />\n" );
+      }
+      myMetadata += QStringLiteral( "<strong>" ) + QObject::tr( "CRS" ) + QStringLiteral( ": </strong>" ) + spatialExtent.extentCrs.authid() + QStringLiteral( " - " );
+      myMetadata += spatialExtent.extentCrs.description() + QStringLiteral( " - " );
+      if ( spatialExtent.extentCrs.isGeographic() )
+        myMetadata += QObject::tr( "Geographic" );
+      else
+        myMetadata += QObject::tr( "Projected" );
+      myMetadata += QStringLiteral( "<br />" );
+      myMetadata += QStringLiteral( "<strong>" ) + QObject::tr( "X Minimum" ) + QStringLiteral( ": </strong>" ) +  qgsDoubleToString( spatialExtent.bounds.xMinimum() ) + QStringLiteral( "<br />" );
```

Comment:
Sidenote, if you add a `Q_DECLARE_TR_FUNCTIONS( QgsLayerMetadataFormatter )` in the class definition this will all be much easier to write.
Sidenote 2: include the `:` puncutation in the string to translate